docs/source/reference/dataset_formats.rst

```diff
@@ -10,7 +10,7 @@ you need a new format, please contact the dsgrid team to discuss it.
 Requirements
 =============
 1. Metric data must be stored in Parquet files. If you need or want another optimized columnar file
-   format, please contact the dsgrid team.
+   format, please contact the dsgrid team. TODO: Describe our (limited) support for CSV?
```

Comment:
We do actually support CSV for datasets. We use them for the AEO growth factor datasets. We _don’t_ want to support them for regular datasets. Maybe we should say that directly?

Reply:
Yes, I think we should say that directly so people won't be confused if they see the .csv datasets in our project repos.